projects/packages/stats/src/class-tracking-pixel.php

```diff
@@ -110,6 +110,58 @@ public static function get_footer_to_add( $data ) {
 		}
 	}
 
+	/**
+	 * Get the list of attributes to add to the Stats script tag.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return string string of attributes to add to the script tag.
+	 */
+	public static function get_script_attributes() {
+		/**
+		 * Allow adding extra attributes to the stats footer script tag.
+		 *
+		 * @module stats
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @param array $script_attributes Array of attributes to add to the script tag. Default to 'defer' only.
+		 */
+		$script_attributes = (array) apply_filters(
+			'jetpack_stats_footer_script_attributes',
+			array( 'defer' => true )
+		);
+
+		/*
+		 * Create an escaped string of attributes to add to the script tag,
+		 * like 'defer', 'async',
+		 * or anything passed to the filter above.
+		 */
+		$script_attributes = implode(
+			' ',
+			array_map(
+				function ( $key, $value ) {
+					/*
+					* For boolean attributes, we either only display the key (e.g. "defer")
+					* or not display anything (e.g. we do not want to display "async=false").
+					*/
+					if ( is_bool( $value ) ) {
+						if ( true === $value ) {
+							return esc_attr( $key );
+						}
+						return; // Skip if the value is false.
```

Comment:
Should probably return the empty string, not null.
```suggestion
						return $value ? esc_attr( $key ) : '';
```
Although it might be even better to do something like this to avoid double-spaces for false values
```php
$tmp = array();
foreach ( $script_attributes as $k => $v ) {
    if ( is_bool( $v ) ) {
        if ( $v ) {
            $tmp[] = esc_attr( $k );
        }
    } else {
        $tmp[] = esc_attr( $k ) . "='" . esc_attr( $v ) . "'";
    }
}
$script_attributes = implode( ' ', $tmp );
```

Reply:
That's a good point, that's more readable that way. Done in dc87b661ef68ae3e8b93312f370c2428fce8862e